src/js/jsx/sections/libraries/Library.jsx

```diff
@@ -76,6 +76,15 @@ define(function (require, exports, module) {
          * @type {AdobeLibraryElement}
          */
         lastLocallyCreatedElement: null,
+        
+        /**
+         * Store the last locally updated graphic element and modified time. Used to determine whether the 
+         * library list should scroll to reveal the updated graphic.
+         * 
+         * @type {AdobeLibraryElement}
+         */
+        lastLocallyUpdatedGraphic: null,
```

Comment:
actually, putting `lastLocallyUpdatedGraphic` in state will trigger unnecessary UI redraw, as the main purpose of these variables are used to decide whether the asset list should scroll after a redraw due to change in `this.props.library`.